Kokoro fails with compilation error

Super-weird compilation failure saying it can't find `java.lang.Object` or `java.lang.String`. It seems like the Kokoro team just updated the Windows image, and that may be related.

```
[ERROR] Failed to execute goal org.eclipse.tycho:tycho-compiler-plugin:1.0.0:compile (default-compile) on project com.google.cloud.tools.eclipse.util: Compilation failure: Compilation failure: 
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[1] 
[ERROR] 	/*
[ERROR] 	^
[ERROR] The type java.lang.Object cannot be resolved. It is indirectly referenced from required .class files
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[1] 
[ERROR] 	/*
[ERROR] 	^
[ERROR] The type java.lang.Class cannot be resolved. It is indirectly referenced from required .class files
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[1] 
[ERROR] 	/*
[ERROR] 	^
[ERROR] The type java.lang.String cannot be resolved. It is indirectly referenced from required .class files
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[24] 
[ERROR] 	private AdapterUtil() {}
[ERROR] 	        ^^^^^^^^^^^^^
[ERROR] Implicit super constructor Object() is undefined. Must explicitly invoke another constructor
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[26] 
[ERROR] 	public static <T> T adapt(Object o, Class<T> cls) {
[ERROR] 	                          ^^^^^^
[ERROR] Object cannot be resolved to a type
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[26] 
[ERROR] 	public static <T> T adapt(Object o, Class<T> cls) {
[ERROR] 	                                    ^^^^^
[ERROR] Class cannot be resolved to a type
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[31] 
[ERROR] 	Object instance = ((IAdaptable) o).getAdapter(cls);
[ERROR] 	^^^^^^
[ERROR] Object cannot be resolved to a type
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\AdapterUtil.java:[36] 
[ERROR] 	Object adapted = Platform.getAdapterManager().loadAdapter(o, cls.getName());
[ERROR] 	^^^^^^
[ERROR] Object cannot be resolved to a type
[ERROR] T:\src\github\google-cloud-eclipse\plugins\com.google.cloud.tools.eclipse.util\src\com\google\cloud\tools\eclipse\util\NatureUtils.java:[1] 
[ERROR] 	/*
  ... (truncated) ...
```

The reason I believe that the Windows image has been updated is that, `gcloud components update` does nothing and the app-engine-java component is already installed at the beginning.

```
T:\src\github\google-cloud-eclipse>set CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true 

T:\src\github\google-cloud-eclipse>call gcloud.cmd components update --quiet 

All components are up to date.

T:\src\github\google-cloud-eclipse>call gcloud.cmd components install app-engine-java --quiet 

All components are up to date.
```

--- a/kokoro/ubuntu/continuous.sh
+++ b/kokoro/ubuntu/continuous.sh
@@ -19,4 +19,4 @@ metacity --sm-disable --replace &
 sleep 3 
 
 cd github/google-cloud-eclipse
-mvn -Ptravis --fail-at-end verify
+mvn -V -Ptravis --fail-at-end verify

--- a/kokoro/windows/continuous.bat
+++ b/kokoro/windows/continuous.bat
@@ -1,4 +1,6 @@
 @echo on
+rem Tycho 1.0.0 does not support Java 9
+set JAVA_HOME="C:\Program Files\Java\jdk1.8.0_152"
 
 rem To speed up build, download and unpack an M2 repo cache.
 pushd %USERPROFILE%

--- a/kokoro/windows/continuous.bat
+++ b/kokoro/windows/continuous.bat
@@ -1,6 +1,6 @@
 @echo on
 rem Tycho 1.0.0 does not support Java 9
-set JAVA_HOME="C:\Program Files\Java\jdk1.8.0_152"
+set "JAVA_HOME=C:\Program Files\Java\jdk1.8.0_152"
 
 rem To speed up build, download and unpack an M2 repo cache.
 pushd %USERPROFILE%

--- a/kokoro/windows/continuous.bat
+++ b/kokoro/windows/continuous.bat
@@ -27,9 +27,9 @@ call gcloud.cmd components update --quiet
 call gcloud.cmd components install app-engine-java --quiet
 @echo on
 
-mvn -B --settings kokoro\windows\m2-settings.xml ^
+mvn -V -B --settings kokoro\windows\m2-settings.xml ^
     -N io.takari:maven:wrapper -Dmaven=3.5.0
-mvnw.cmd -B --settings kokoro\windows\m2-settings.xml ^
+mvnw.cmd -V -B --settings kokoro\windows\m2-settings.xml ^
          --fail-at-end -Ptravis -Declipse.target=oxygen verify
 
 exit /b %ERRORLEVEL%

